tensorboard/webapp/settings/_views/settings_module.ts

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
This file's naming scheme is misaligned with notifications_center/'s.

- `notification_center/_views/views_module.ts`
- `settings/_views/settings_module.ts`

Which naming scheme is correct?

Reply:
There was no prescription around that. I'm leaning towards latter (thus my PR :D)